content/id/docs/tasks/job/automated-tasks-with-cron-jobs.md

```diff
@@ -0,0 +1,214 @@
+
+Sekarang, temukan Pod yang dibuat oleh jadwal Job terakhir dan melihat keluaran bawaan dari salah satu Pod
+
+{{< note >}}
+Nama Job dan nama Pod itu berbeda.
+{{< /note >}}
+
+```shell
+# Ganti "hello-4111706356" dengan nama Job di sistem kamu
+pods=$(kubectl get pods --selector=Job-name=hello-4111706356 --output=jsonpath={.items[*].metadata.name})
+```
+Menampilkan log sebuah Pod:
+
+```shell
+kubectl logs $pods
+```
+Keluaran akan mirip dengan ini:
+
+```
+Fri Feb 22 11:02:09 UTC 2019
+Hello dari klaster Kubernetes
+```
+
+## Menghapus sebuah CronJob
+
+Ketika kamu tidak membutuhkan sebuah CronJob lagi, kamu dapat megnhapusnya dengan perintah `kubectl delete cronJob <nama cronJob>`:
+
+```shell
+kubectl delete cronJob hello
+```
+
+Menghapus CronJob akan menghapus semua Job dan Pod yang telah terbuat dan menghentikanya dari pembuatan Job tambahan.
+Kamu dapat membaca lebih lanjut tentang menghapus Job di [_garbage collection_](/docs/concepts/workloads/controllers/garbage-collection/).
+
+## Menulis Speifikasi Sebuah Cron
+
+Seperti semua konfigurasi Kubernetes, sebuah CronJob membutuhkan bidang `apiVersion`, `kind`, dan `metadata`. Untuk informasi
+umum tentang bekerja dengan berkas konfigurasi, lihat dokumentasi [men-deploy aplikasi](/docs/user-guide/deploying-applications),
+dan [mengunakan kubectl untuk manajemen sumber daya](/docs/user-guide/working-with-resources).
+
+Sebuah konfigurasi CronJob juga membutuhkan sebuah [bagian `.spec`](https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status).
+
+{{< note >}}
+Semua modifikasi pada sebuah CronJob, terutama `.spec`, akan diterapkan pada proses berikut.
+{{< /note >}}
+
+### Penjadwalan
+
+`.spec.schedule` adalah bidang yang wajib diisi dari sebuah `.spec`
+Dibutuhkan sebuah fromat string [Cron](https://en.wikipedia.org/wiki/Cron), misalnya `0 * * * *` atau `@hourly`, sebagai jadwal Job untuk dibuat dan dieksekusi.
+
+Format ini juga mencakup nilai langkah `vixie cron`. Seperti penjelasan di [FreeBSD manual](https://www.freebsd.org/cgi/man.cgi?crontab%285%29):
+
+> Nilai langkah dapat digunakan bersama dengan rentang. Sebuah rentang diikuti dengan
+> `/<number>` menentukan lompatan angka melalui rentang. 
+> Misalnya, `0-23/2` dapat digunakan dalam jam untuk menentukan
+> perintah akan dieksekusi setiap jam (alternatif dalam bawaan v7 adalah
+> `0,2,4,6,8,10,12,14,16,18,20,22`). Langkah-langkah juga diizinkan setelah
+> tanda bintang, jadi jika kamu menginginkan "setiap dua jam", gunakan saja `*/2`.
+
+{{< note >}}
+Sebuah tanda tanya (`?`) dalam penjadwalan memiliki makna yang sama dengan tanda bintang `*`, yaitu, singkatan dari nilai apa pun yang tersedia untuk bidang tertentu.
+A question mark (`?`) in the schedule has the same meaning as an asterisk `*`, that is, it stands for any of available value for a given field.
```

Comment:
This sentence is still English

Reply:
forgot to delete the english part, will fix that